src/test/throttler/throttler.spec.ts

```diff
@@ -3,14 +3,20 @@ import * as sinon from "sinon";
 
 const { expect } = chai;
 
-import Queue from "../queue";
+import Queue from "../../throttler/queue";
+import Stack from "../../throttler/stack";
 
 const TEST_ERROR = new Error("foobar");
 
-describe("Queue", () => {
+const throttlerTest = (Throttler: any) => {
```

Comment:
Any good suggestion for `Throttler`'s type?

Reply:
`Stack | Queue`?

Reply:
No, it won't work because `Throttler` is not an instance of Stack or Queue. It's the classOf them.

Reply:
ah! you'll want to see this part of the documentation: https://www.typescriptlang.org/docs/handbook/interfaces.html and scroll down to "Class Types"